### ISSUE TITLE
RMS support nested comment, but syntax highlight doesn't

### ISSUE THREAD

**Issue by fntsrlike** [mangudai/vscode#2]
## Info
- VS Code version:  `1.16.1`
- Extension version: `0.2.9`

## Description
In RMS, it  support nested comment (though it confuse me, but it does). Syntax highlight should also support it, or developers will be bothered when they use nested comment accidentally.

### Except
```
/*
 * Comment
/*
 * Nested Comment
 */

SOME INSTRUCTION...
```

Because the nested comment doesn't be close, the code after comment should also be comment's syntax highlight.

### Actually
As above, the comment's syntax highlight will be stop on the first comment closing.

#### Screenshot
On below, it should be green at all as it not really closing comment.
![2017-09-24 8 14 21](https://user-images.githubusercontent.com/2680724/30778144-6730fa38-a100-11e7-84c6-534795064660.png)

On below, it should be green at all, and stop on the second closing syntax.
![2017-09-24 8 12 46](https://user-images.githubusercontent.com/2680724/30778135-2d4839e4-a100-11e7-9195-e3d176e5b68e.png)

**Comment by deltaidea** [comment]
I've dreaded the day someone will need this feature. I'll look into it. Thanks for the report! ❤️

**Comment by deltaidea** [comment]
Fixed in 0.3.3.
